packages/amplify-category-geo/src/service-utils/resourceParams.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Parameters common to geo resources
+ */
+export type ResourceParameters = {
+    providerContext: ProviderContext,
+    name: string,
+    pricingPlan: PricingPlan,
+    accessType: AccessType,
+    isDefault: boolean,
+    dataProvider: DataProvider
+}
+
+/**
+ * higher level context
+ */
+export interface ProviderContext {
```

Comment:
This type probably shouldn't go here. `amplify-cli-core/src/index` might be a better place for it. That file has kinda turned into a dumping ground for lots of platform types, but until we have a better organization strategy, that's probably the best place for it